### PR TITLE
Add link to previous speaking videos on speaker profile.

### PR DESCRIPTION
**Pull request by RageZBla**
References #151

**Proposed Changes**

Community has asked to have link to video recording of previous presentation for the speaker. This would help the speakers to get picked.

**Testing Plan**

>>>
Go to the nominate speaker form, try to submit without the link to previous presentation. 
Try to submit with the link.
Check if the speaker has a link to the submitted link with the YouTube link
>>>

### CODE REVIEW

File: web/src/components/speaker/Links.vue [hunk 1]
```diff
@@ -13,6 +13,9 @@
       <!--eslint-disable-next-line @intlify/vue-i18n/no-raw-text -->
       <v-icon>web</v-icon>
     </social-button>
+    <social-button :link="priorPresentation">
+      <i class="fab fa-youtube" />
```
Comment (ann-kilzer) [inline web/src/components/speaker/Links.vue:17]: We don't want to box ourselves into youtube, as event organizers might also upload to vimeo or other platforms. Let's use the material icon [ondemand_video](https://material.io/resources/icons/?icon=ondemand_video&style=baseline).

```suggestion
       <!--eslint-disable-next-line @intlify/vue-i18n/no-raw-text -->
      <v-icon>ondemand_video</v-icon>
```

File: web/src/components/nomination/NominateForm.vue [hunk 2]
```diff
@@ -224,6 +227,7 @@ export default {
           twitter: '',
           facebook: '',
           website: '',
+          priorPresentation: '',
```
Comment (ann-kilzer) [inline web/src/components/nomination/NominateForm.vue:230]: We'll need to add this to the payloadFields on line 381 so it gets sent to Airtable. I'll go ahead and create a new column `prior_presentation_url`.

**Review by ann-kilzer — COMMENTED** [review]
Thanks for picking this up. Left some brief feedback.

File: web/src/components/nomination/NominateForm.vue (hunk 2 above)
Comment (RageZBla) [reply web/src/components/nomination/NominateForm.vue:230]: @ann-kilzer Thanks I have overlooked that bit.

**Comment by RageZBla** [comment]
@ann-kilzer I have addressed your feedback 🙇

File: web/src/components/speaker/Links.vue [hunk 3]
```diff
@@ -40,6 +44,10 @@ export default {
       type: String,
       default: '',
     },
+    priorPresentation: {
```
Comment (ann-kilzer) [inline web/src/components/speaker/Links.vue:47]: We'll need to pass this prompt in on the <links> component from the parent as well.

**Review by ann-kilzer — COMMENTED** [review]
Very close! I tested submission and it works, but I don't see the video link icon showing up yet. I added sample data to my own record "Ann Kilzer" so that's a good one to inspect for testing.

File: web/src/components/speaker/Links.vue (hunk 3 above)
Comment (RageZBla) [reply web/src/components/speaker/Links.vue:47]: @ann-kilzer thanks for catching this and my apologies for the back and forth.

**Comment by RageZBla** [comment]
@ann-kilzer hopefully this should be good to merge 🤞

**Comment by tuttiq** [comment]
@RageZBla Thank you for this PR!

Screen Shot 2020-10-04 at 12 17 39 AM

Can we make the input full-width to avoid the blank space without any input beside it?

**Comment by RageZBla** [comment]
@tuttiq I have updated the layout as per feedback 🙇

**Review by tuttiq — APPROVED** [review]
LGTM! 👍

**Pull request merged**